src/api/projects.js

```diff
@@ -1,60 +1,99 @@
 const express = require('express');
 const projetsExample = require('./dev/projectsExamples.json');
+const prisma = require('../../prismaClient');
 
 const router = express.Router();
 
 router.get('/', (req, res) => {
-  res.status(200).json(projetsExample);
+  prisma.project
+    .findMany()
+    .then((projects) => {
+      res.status(200).json(projects);
+    })
+    .catch((error) => {
+      res.status(500).json(error);
+    });
 });
 
-router.get('/:id', (req, res) => {
```

Comment:
Then/catch in /projects route, async/await in /projects/:id. I think we should choose and use same method in all the ressource.